apps/react-workshop/src/Breadcrumbs.stories.tsx

```diff
@@ -117,39 +117,49 @@ export const CustomOverflowBackButton = () => {
     ));
 
   return (
-    <div
-      style={{
-        width: '50%',
-        maxWidth: 725,
-        minWidth: 150,
-        border: '1px solid lightpink',
-        padding: 8,
+    <Breadcrumbs
+      overflowButton={(visibleCount: number) => {
+        const previousBreadcrumb =
+          visibleCount > 1 ? items.length - visibleCount : items.length - 2;
+        return (
+          <Tooltip content={`Item ${previousBreadcrumb}`} placement='bottom'>
```

Comment:
Since you're already in this file, maybe you can replace the Tooltip and aria-label with the `label` prop on IconButton